renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -739,6 +742,50 @@ std::string WrappedOpenGL::GetChunkName(uint32_t idx)
   return ToStr((GLChunk)idx);
 }
 
+ResourceId WrappedOpenGL::GetContextResourceID()
+{
+  uint64_t currentId = Threading::GetCurrentID();
+  if(m_ActiveContextResourceIDs.find(currentId) == m_ActiveContextResourceIDs.end())
+  {
+    m_ActiveContextResourceIDs[currentId] =
+        GetResourceManager()->RegisterResource(GLResource(NULL, eResSpecial, eSpecialResContext));
+  }
+  return m_ActiveContextResourceIDs[currentId];
+}
+
+GLResourceRecord *WrappedOpenGL::GetContextRecord()
+{
+  ResourceId currentResourceId = GetContextResourceID();
+  if(!GetResourceManager()->HasResourceRecord(currentResourceId))
+  {
+    GLResourceRecord *contextRecord = GetResourceManager()->AddResourceRecord(currentResourceId);
+    contextRecord->DataInSerialiser = false;
+    contextRecord->Length = 0;
+    contextRecord->SpecialResource = true;
+  }
+  return GetResourceManager()->GetResourceRecord(currentResourceId);
+}
+
+bool WrappedOpenGL::RecordingHasChunks()
```

Comment:
This function can actually just be removed, the check it's replacing is a hangover from D3D11's backend - the `m_ContextRecord` will always have at least two chunks regardless.